airflow/dag_processing/manager.py

```diff
@@ -676,7 +679,7 @@ def _fetch_callbacks(self, max_callbacks: int, session: Session = NEW_SESSION):
     @retry_db_transaction
     def _fetch_callbacks_with_retries(self, max_callbacks: int, session: Session):
         """Fetch callbacks from database and add them to the internal queue for execution."""
-        self.log.debug("Fetching callbacks from the database.")
+        self.log.warning("Fetching callbacks from the database.")
```

Comment:
Why do we change this to a warning?

Reply:
I was testing these as the PR was in draft. I will change this.